src/argilla/client/feedback/dataset.py

```diff
@@ -489,7 +489,7 @@ def push_to_argilla(self, name: Optional[str] = None, workspace: Optional[Union[
         """
         httpx_client: "httpx.Client" = rg.active_client().http_client.httpx
 
-        if not name or (not name and not workspace):
+        if name is None:
```

Comment:
With this change, the warning message looks outdated. Do we still need to check the workspace argument?

Reply:
no, we don't. I'll update the warning message